### ISSUE TITLE
MacOS - CommandExecutionError: Service not found: docker

### ISSUE THREAD

**Issue by noelmcloughlin** [saltstack-formulas/docker-formula#210]
Fails on MacOS
```
ID: docker-service
    Function: service.running
        Name: docker
      Result: False
     Comment: An exception occurred in this state: Traceback (most recent call last):
                File "/usr/local/Cellar/salt/2019.2.0_1/libexec/lib/python3.7/site-packages/salt/state.py", line 1933, in call
                  **cdata['kwargs'])
                File "/usr/local/Cellar/salt/2019.2.0_1/libexec/lib/python3.7/site-packages/salt/loader.py", line 1939, in wrapper
                  return f(*args, **kwargs)
                File "/usr/local/Cellar/salt/2019.2.0_1/libexec/lib/python3.7/site-packages/salt/states/service.py", line 928, in mod_watch
                  __salt__['service.stop'](name)
                File "/usr/local/Cellar/salt/2019.2.0_1/libexec/lib/python3.7/site-packages/salt/modules/mac_service.py", line 439, in stop
                  domain_target, path = _get_domain_target(name)
                File "/usr/local/Cellar/salt/2019.2.0_1/libexec/lib/python3.7/site-packages/salt/modules/mac_service.py", line 208, in _get_domain_target
                  service = _get_service(name)
                File "/usr/local/Cellar/salt/2019.2.0_1/libexec/lib/python3.7/site-packages/salt/modules/mac_service.py", line 142, in _get_service
                  raise CommandExecutionError('Service not found: {0}'.format(name))
              salt.exceptions.CommandExecutionError: Service not found: docker
```

**Comment by noelmcloughlin** [comment]
Only happens on initial install maybe.
```
          ID: docker-package
    Function: pip.installed
        Name: pip
      Result: True
     Comment: Python package pip was already installed
              All specified packages are already installed
     Started: 21:10:40.773589
    Duration: 672.544 ms
     Changes:   
----------
          ID: docker-service
    Function: service.running
        Name: docker
      Result: False
     Comment: The named service docker is not available
     Started: 21:10:41.447075
    Duration: 225.912 ms
     Changes:   
----------
          ID: docker-py
    Function: pip.installed
        Name: docker
      Result: True
     Comment: Python package docker was already installed
              All specified packages are already installed

```

**Comment by noelmcloughlin** [comment]
Resolved by #222
